clang/test/CodeGen/catch-nullptr-and-nonzero-offset.c

```diff
@@ -225,172 +219,6 @@ char *nullptr_allones_BAD(void) {
 
 //------------------------------------------------------------------------------
 
-char *one_var(unsigned long offset) {
```

Comment:
After spending way too much time updating manually updating these tests, I decided to delete them altogether. I don't think they add any value over the generic tests (using non-constants) higher up, and just end up testing details of IRBuilder's constant folding behavior.